cpp/tensorrt_llm/kernels/cutlass_kernels/moe_gemm/moe_kernels.cu

```diff
@@ -1360,6 +1363,10 @@ void expandInputRowsKernelLauncher(InputActivationsType const* unpermuted_input,
         check_cuda_error(cudaMemsetAsync(
             fc1_act_sf_flat, 0x0, num_elems * sizeof(TmaWarpSpecializedGroupedGemmInput::NVFP4ElementSF), stream));
     }
+    else
+    {
+        TLLM_CHECK_WITH_INFO(!use_per_expert_act_scale, "Per-expert act scale is only supported for FP4 activations");
```

Comment:
This error msg is a bit confusing. What quant schemes does per-expert act scale support? I thought it supports:
- fp8 per-tensor
- nvfp4
- fp8/mxfp4 w4a8